studies/models.py

```diff
@@ -127,7 +131,7 @@ class Study(models.Model):
     shared_preview = models.BooleanField(default=False)
     creator = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
 
-    metadata = DateTimeAwareJSONField(default={})
+    metadata = DateTimeAwareJSONField(default=dict)
```

Comment:
Oh wow surprised that wasn't a bug earlier - never use mutable objects as defaults on keyword args, otherwise the reference stays the same because the dict literal is evaluated at function parse time. That's why they make you use factory funcs

Reply:
Yep, that's a surprise. At least we'd caught the other instances of this.